index.html

```diff
@@ -1430,9 +1441,48 @@
 
 ## Harassment
 
-<aside class="issue">
-  This section has not been written.
-</aside>
+Online <dfn>harassment</dfn> is the "pervasive or severe targeting of an individual or group online
+through harmful behavior" [[PEN-Harassment]]. Harassment is a prevalent problem on the Web,
+particularly via social media. Harassment may be more severe and its consequences more impactful for
+LGBTQ people and women [[?PEW-Harassment]].
+
+<a>Harassment</a> is both a violation of privacy itself and can be a consequence of other violations
+of privacy. 
+
+Abusive online behavior may include: sending <a href="#unwanted-information">unwanted
+information</a>; directing others to contact or bother a person ("dogpiling"); disclosing sensitive
+information about a person; posting false information about a person; impersonating a person;
+insults; threats; and hateful or demeaning speech.
+
+Disclosure of contact information can be used, including by a separate attacker, to send often
+persistent unwanted information that amounts to harassment. Disclosure of location information can
+be used, including by a separate attacker, to intrude on a person's physical safety or space.
+
+Mitigations for harassment include but extend beyond mitigations for unwanted information and other
+privacy principles. Harassment can include harmful activity not solely directly communicated to the
+target of harassment.
+
+<div class="practice">
+  <p>
+    <span class="practicelab" id="abuse-reporting">
+      Systems that allow for communicating on the Web must provide an effective capability to report
+      abuse.
```

Comment:
Reporting _and_ enforcement, no?

Reply:
That's a good topic for broader discussion. I do think some enforcement is needed in order for the reporting abuse mitigation to be effective. I'm less clear on how specifically we can give guidance on that (or how often that would be in the standards or web implementations discussed at w3c), but I would welcome a text suggestion on that.

Reply:
I don't think that we could give good guidance on enforcement at this stage, but I think that keeping enforcement implicit behind "effective" is a bit too indirect. Maybe "must provide an effective capability to report abuse coupled with forceful enforcement."

Reply:
We agreed to leave enforcement out for now in the call today. Can revisit in future.